velox/docs/configs.rst

```diff
@@ -697,6 +697,13 @@ Spark-specific Configuration
      - integer
      -
      - The current task's Spark partition ID. It's set by the query engine (Spark) prior to task execution.
+   * - spark.legacy_time_parser_policy
+     - bool
+     - false
+     - If true, simple date formatter is used for time formatting and parsing. Joda date formatter is used by default.
```

Comment:
It might be helpful to capitalize Simple and make it a link to Javadocs. Otherwise, readers may be confused thinking "simple" means literally something simple.

Reply:
Thanks. Done.